demo/NeMo/README.md

```diff
@@ -0,0 +1,135 @@
+# TensorRT FP8 Inference for NeMo models
+This repository demonstrates TensorRT inference with NeMo Megatron models in FP8/FP16/BF16 precision.
+
+Currently, this repository supports [NeMo GPT](https://huggingface.co/nvidia/nemo-megatron-gpt-5B) models only.
+
+# Environment Setup
+It's recommended to run inside a container to avoid conflicts when installing dependencies. Please check out [`NGC TensorRT`](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorrt/tags) and find a container with TensorRT 9.0 or above. A GPU with compute capability 8.9 or above is required to run the demo with FP8 precision.
```

Comment:
```suggestion
It's recommended to run inside a container to avoid conflicts when installing dependencies. Please check out [NGC TensorRT](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorrt/tags) and find a container with TensorRT 9.0 or above. A GPU with compute capability 8.9 or above is required to run the demo with FP8 precision.
```